.travis.yml

```diff
@@ -7,5 +7,8 @@ matrix:
   allow_failures:
   - node_js: node
 
+before_script:
+  - make develop & sleep 20
```

Comment:
You probably want to use a double ampersand, so if `make develop` fails, you don't have to wait 20 seconds to find out

Reply:
The command times out when using double ampersand